### PR TITLE
Packaging changes to address 8176 and glTF example file copy / material import errors

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change updates the file layout in the foundation, extensions and examples packages to resolve:

- #8176 
- Observed failure to copy files to StreamingAssets folder (glTF demo)
- Error when attempting to import material in the included avocado model

### CODE REVIEW

File: Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ namespace Microsoft.MixedReality.Toolkit.Examples.Demos.Gltf.Editor
     [CustomEditor(typeof(TestGltfLoading))]
     public class TestGltfLoadingEditor : UnityEditor.Editor
     {
-        private static readonly string GLTFModelsPath = $"Demos{Path.DirectorySeparatorChar}Gltf{Path.DirectorySeparatorChar}Models";
+        private static readonly string GLTFModelsPath = "Common/Gltf/Models";
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs:17]: To keep the original formatting?:

```suggestion
        private static readonly string GLTFModelsPath = $"Common{Path.DirectorySeparatorChar}Gltf{Path.DirectorySeparatorChar}Models";
```

**Review by MaxWang-MS — APPROVED** [review]
Looks good to me!

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs [hunk 2]
```diff
@@ -29,11 +31,10 @@ public override void OnInspectorGUI()
                 EditorGUILayout.HelpBox("glTF path was not discovered in the streaming assets folder. Please copy over files to test example scene", MessageType.Warning);
             }
 
-            if (GUILayout.Button("Copy GLTF Directory to Streaming Assets"))
+            if (GUILayout.Button("Copy models to StreamingAssets"))
             {
-                string modelPath = MixedRealityToolkitFiles.MapRelativeFolderPathToAbsolutePath(MixedRealityToolkitModuleType.Examples, GLTFModelsPath);
-                DirectoryCopy(modelPath, Path.Combine(Application.streamingAssetsPath, "GltfModels"));
-                AssetDatabase.Refresh(ImportAssetOptions.ForceUpdate);
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs:36]: Should this line be restored? Feels necessary

Comment (davidkline-ms) [reply Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs:36]: it's back

File: Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs [hunk 3]
```diff
@@ -3,8 +3,10 @@
 
 using Microsoft.MixedReality.Toolkit.Utilities.Editor;
 using System.IO;
+using System.Linq;
 using UnityEditor;
 using UnityEngine;
+using UnityEngine.Experimental.GlobalIllumination;
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs:9]: Are these new `using`s used?

Comment (davidkline-ms) [reply Assets/MRTK/Examples/Demos/Gltf/Scripts/Editor/TestGltfLoadingEditor.cs:9]: i do not believe so... i will remove them

**Review by keveleigh — APPROVED** [review]

**Pull request merged**